src/payloads/features/idpay/get-timeline-detail.ts

```diff
@@ -21,19 +21,13 @@ const generateRandomOperationDetailDTO = (
         endDate: faker.date.recent(),
         transferDate: faker.date.recent()
       };
-    case "TRANSACTION":
-      return {
-        ...operation,
-        operationType: TransactionDetailEnum.TRANSACTION,
-        idTrxAcquirer: ulid(),
-        idTrxIssuer: ulid()
-      };
     case "REVERSAL":
+    case "TRANSACTION":
       return {
         ...operation,
-        operationType: TransactionDetailEnum.REVERSAL,
         idTrxAcquirer: ulid(),
-        idTrxIssuer: ulid()
+        idTrxIssuer: ulid(),
+        status: operation.status as unknown as StatusEnum // OperationListDTO[StatusEnum] to OperationDTO[StatusEnum] conversion
```

Comment:
```suggestion
        status: operation.status
```
tested with the new definitions and there seems to be no need, most likely since types now overlap

Reply:
idpay-full from [this pr](https://github.com/pagopa/cstar-infrastructure/pull/949)'s branch

Reply:
Will be fixed in next PR